checkov/cloudformation/runner.py

```diff
@@ -61,7 +61,7 @@ def run(
             self.context = build_definitions_context(self.definitions, self.definitions_raw, root_folder)
 
             logging.info("creating cloudformation graph")
-            local_graph = self.graph_manager.build_graph_from_definitions(self.definitions)
+            local_graph = self.graph_manager.build_graph_from_definitions(self.definitions, render_variables=True)
```

Comment:
@Saarett Why not change the default value of the param in the `build_graph_from_definitions` method definition from false to true?